psi4/driver/procedures/proc.py

```diff
@@ -2172,8 +2172,14 @@ def run_ccenergy(name, **kwargs):
     if core.get_global_option('RUN_CCTRANSORT'):
         core.cctransort(ref_wfn)
     else:
-        core.transqt2(ref_wfn)
-        core.ccsort()
+        # try:
+        from psi4.driver.pasture import addins
```

Comment:
As `RUN_CCTRANSORT` needs to stay valid, we probably need the try/except block to prevent possible seg faults or weird errors. This sort of area should be the only entry point for the addins.